### PR TITLE
Add initial documentation for the Pulumi Service API

### PR DESCRIPTION
**Pull request by clstokes**
Adds initial documentation for the Pulumi Service API.

### CODE REVIEW

File: content/docs/reference/service-api/_index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+| --------- | ---------- | ---------- |
+| `role` | string | role to assign - possible values are `admin` or `member` |
+
+#### Sample Payload
+
+```
+{ "role": "{role}" }
+```
+
+#### Curl Example
```
Comment (susanev) [inline content/docs/reference/service-api/_index.md:94]: i wonder if this should be closer to the top?

**Comment by davidwrede** [comment]
@clstokes Got sidetracked with other work. Sorry about that. I agree with @infin8x on all his main points. Here's some additional feedback:

- The heading for the CURL example can just be "Example". If/when we have examples using languages, then I think that's when it makes sense to qualify the example (or keep the "Example" heading and use our language chooser).
- On the Request body parameters, we should also denote if they are required or optional. You can either add in a Required/Optional column, or just fold that into the "Name" or "Description" columns.

**Comment by clstokes** [comment]
@infin8x @davidwrede, I incorporated your comments into my latest commit. Give it another look?

http://pulumi-docs-origin-pr-5338-69ae0ef1.s3-website.us-west-2.amazonaws.com/docs/reference/service-api/

**Review by davidwrede — APPROVED** [review]

**Comment by cnunciato** [comment]
@clstokes Now that https://github.com/pulumi/docs/pull/5494 has merged, would you mind resubmitting this on https://github.com/pulumi/pulumi-hugo? Sorry the the inconvenience and happy to help if you need.

**Comment by clstokes** [comment]
Will move this to the other repo. Closing.

**Pull request closed without merging**